### ISSUE TITLE
Blue cursor in code cells

### ISSUE THREAD

**Issue by markphilpot** [HappenApps/Quiver-iOS#21]
There is a blue cursor in code cell (I was in bash) and it would be offset (a few characters off) from the blinking white cursor.

Sometimes this cursor had blue circles on top and bottom, and sometimes it didn't.  These might have been handles to select text, but they didn't seem to be active (did not respond to touch)

**Comment by nekonora** [comment]
As far as I tried it happens in markdown cells too.

**Comment by ylian** [comment]
Yeah, it's an issue with the Ace editor. I need to make some significant changes to Ace to make it work well on mobile.

**Comment by kerim** [comment]
This is also a problem when editing the CSS for styles in the settings.
